tests/hook_implementations/test_hook_implementations.py

```diff
@@ -30,7 +33,6 @@ def test_hook_implementations():
     installable_workflow_jobs = pm.get_installable_workflow_jobs()
     for wf_name, wf_location in expected_workflow_jobs.items():
         assert wf_name in installable_workflow_jobs
```

Comment:
wf_location may be unnecessary.

Reply:
Good point, because the exact path will now vary between runs (and is the domain of `ert`, not semeio), I've changed to asserting that it is a file instead.